tests/Unit/ErrorHandling/Test_FloatingPointExceptions.cpp

```diff
@@ -8,21 +8,10 @@
 
 #include "Utilities/ErrorHandling/FloatingPointExceptions.hpp"
 
-// Trapping floating-point exceptions is apparently unsupported on
-// the arm64 architecture, so when building on Apple Silicon,
-// directly call the fpe_signal_handler in these tests so that they pass.
```

Comment:
Is this no longer true?

Reply:
looks like I screwed up reverting the file after trying to convert these tests to use `CHECK_THROW_WITH` unsuccessfully